front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx

```diff
@@ -204,6 +205,14 @@ const ProjectActionButtons = memo<Props>(({ projectId, className }) => {
             fontWeight="500"
           />
         )}
+      {isParticipationMethodNativeSurvey && (
+        <IdeaButton
+          id="project-ideabutton"
```

Comment:
Suggestion: Non-blocking. 

Looking at [this](https://github.com/CitizenLabDotCo/citizenlab/blob/cl-1253-refactor-ideas-add-survey-handling/front/app/containers/ProjectsShowPage/shared/header/ProjectActionButtons.tsx#L190-L215) block of code, it looks like the three conditions can exist at the same time which I don't think might be the case practically. Some of the props passed on to the `IdeaButton` component are similar and the differences are only on `phaseId` and `participationContextType`. That being known, we could use one component and change what we pass in based on condition. The advantage is that the code would explicitly show that only one of these is rendered at a time.

Down the line, we could also rename the id to something different